data-raw/qc-markdowns/temperature_data_prep.Rmd

```diff
@@ -228,14 +228,14 @@ write_rds(BCK_daily_temps, here::here("data-raw", "standard-format-data-prep","t
 
 ### Recommendations
 
--   CDEC data has very high max - I think these are errors or code for
-    something. CDEC data is more similar to Upper Clear Creek
--   Use the data from Clear over the CDEC data.
```

Comment:
These notes we had are better than nothing but forming this into more coherent statements about rationale would be helpful! I made some edits and pushed these for reference.